pkg/client/client.go

```diff
@@ -227,6 +228,40 @@ func (d *DataCenterClient) GetUsers(ctx context.Context, pToken *pagination.Toke
 	return userData.Users, nextPageToken, err
 }
 
+func (d *DataCenterClient) GetGroupUsers(ctx context.Context, group string) ([]User, error) {
+	var users []User
+	var err error
+	start := 0
+
+	for {
+		queryParams := map[string]string{
+			"start": strconv.Itoa(start),
+			"limit": strconv.Itoa(ITEMSPERPAGE),
+			"group": group,
+		}
+
+		uri, err := d.MakeURL(ctx, allUsersEndpoint, queryParams)
+		if err != nil {
+			return nil, err
+		}
+
+		var userData UsersAPIData
+		resp, err := d.Do(ctx, http.MethodGet, uri, nil, &userData)
+		if err != nil {
+			return nil, err
+		}
+		defer resp.Body.Close()
```

Comment:
this defer isn't safe inside the for loop?

Reply:
Whoops. I close the response body every time now.

I'm guessing I didn't run into any bugs testing because baton-SDK's http wrapper replaces the response body with a noop reader.